.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,69 @@
+---
+name: Feature Request
+about: "In order to build better products, software developers need user feedback. In the very end, it is you that SeAT is about and should enable you to spend more time in game. For us developers you are a great source of inspiration and ideas. If you have an idea or a suggestion to improve seat even more use this template.
+---
+
+# Feature Request
+*This template contains an example feature request. Please replace all text except for the checklist and the section headers (they start with \#). Thank you for your contribution to improving SeAT.*
+
+*In this space, please provide a short description of your feature request.
+This should help us to understood what you'd like to be able to do. E.g.:*
+
+It would be great having a corporation Dashboard for Directors/CEOs as home-screen containing following information:
+
+- Compliance stats (#members with valid tokens and token changes),
```

Comment:
is the # a typo ? or related to "number of" ?

Reply:
number of